modules/contentbox/models/system/CBHelper.cfc

```diff
@@ -369,7 +369,7 @@ component accessors="true" singleton threadSafe{
 
 		// Do we set a layout for them already?
 		if( structKeyExists( arguments, "layout" ) ){
-			event.setLayout( name="#prc.cbTheme#/layouts/" & arguments.layout, module="contentbox" );
+			event.setLayout( name="#prc.cbThemeRecord.includePath#/layouts/" & arguments.layout );
```

Comment:
I am not sure this will work.  Just use :`setLayout( name="", module="" )` instead

Reply:
it works fine... i tested it on my lucee.
Should i use module equal nothing?

Reply:
Why would the module be empty?

Reply:
If you set a module, it looks in that module for the layout. 
So leaving it blank, seems to look in the root of the app.

What do you think?

Reply:
Ok so I am confused then. Are you saying you want it to look in the root? Or in a module?